src/main/java/lotto/model/Receipt.java

```diff
@@ -0,0 +1,44 @@
+package lotto.model;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class Receipt {
+    private final Lotteries auto;
+    private final Lotteries manual;
+
+    public Receipt(Lotteries auto, Lotteries manual) {
+        this.auto = auto;
+        this.manual = manual;
+    }
+
+    public int getAutoGameCount() {
+        return auto == null ? 0 : auto.size();
+    }
+
+    public int getManualGameCount() {
+        return manual == null ? 0 : manual.size();
+    }
+
+    public Lotteries getLotteries() {
+        List<Lottery> lotteries = new LinkedList<>();
+        if (getManualGameCount() > 0) {
+            lotteries.addAll(manual.getLotteries());
+        }
+        lotteries.addAll(auto.getLotteries());
+        return new Lotteries(lotteries);
+    }
+
+    public List<Lottery> getAllLottery() {
+        List<Lottery> lotteries = new LinkedList<>();
+        if (getManualGameCount() > 0) {
+            lotteries.addAll(manual.getLotteries());
+        }
+        lotteries.addAll(auto.getLotteries());
+        return lotteries;
+    }
+
+    public Money getMoney() {
+        return new Money((getAutoGameCount() + getManualGameCount() * 1000));
```

Comment:
```suggestion
        return new Money((getAutoGameCount() + getManualGameCount() * Ticket.UNIT));
```
으로 작성해줘도 좋을 것 같습니다 : )